dashboard/routings/export_data.py

```diff
@@ -12,10 +14,9 @@
 @admin_secure
 def download_csv():
     csv = "\"ENDPOINT\",\"EXECUTION_TIME\",\"TIME_OF_EXECUTION\",\"VERSION\",\"GROUP_BY\",\"IP_ADDRESS\"\n"
-    data = get_data()
-    for entry in data:
-        csv += "\"{0}\",{1},\"{2}\",\"{3}\",\"{4}\",\"{5}\"\n".format(entry.endpoint, entry.execution_time, entry.time,
-                                                                      entry.version, entry.group_by, entry.ip)
+    for entry in get_data():
+        csv += "\"{0}\",{1},\"{2}\",\"{3}\",\"{4}\",\"{5}\"\n".format(
```

Comment:
you're aware of the Python3 f`{entry.endpoint},{entry.execution}...` syntax, right? :)